src/main/webapp/resources/js/listOrders.js

```diff
@@ -0,0 +1,40 @@
+function listAllOffers() {
+    var request = new XMLHttpRequest();
+    var url = "/offers";
+    request.open("GET", url, true);
+
+    request.onload = function() {
+        var offersList = JSON.parse(request.responseText);
+        if(this.readyState == 4 && this.status == 200) {
+             for(var i = 0; i < offersList.length; i++) {
+                listOffer(offersList[i])
+             }
+        }
+    }
+    request.send();
+}
+
+function listOffer(offer) {
+    var imagePath = 'https://placehold.it/150x80?text=IMAGE';
+    var objTo = document.getElementById('main-container')
+    var offerDiv = document.createElement("div");
+    offerDiv.className = "col-sm-4";
```

Comment:
.className is an attribute for divs (and for other HTML tags). It will produce tag like:
<div class="col-sm-4">